### PR TITLE
Extend Tiger user and workspace functionality

### PR DESCRIPTION
**Pull request by Tulean**
JIRA: RAIL-3039
JIRA: RAIL-3016

---

Supported PR commands:

| Command                | Description            |
| ---------------------- | ---------------------- |
| `ok to test`           | Re-run standard checks |
| `extended test`        | BackstopJS tests       |
| `extended check sonar` | SonarQube tests        |

---

# PR Checklist

-   [ ] commit messages adhere to the [commit message guidelines](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#what-should-the-commits-look-like)
-   [ ] review was done by a Code owner [if necessary](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-tell-if-my-pull-request-needs-approval-by-a-code-owner) (if you think it is not necessary, explain the reasoning in the description or in a comment)
-   [x] `check` passes
-   [x] `check-extended` passes
-   [ ] `rush change` [was run if applicable](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-describe-my-changes-for-the-changelog)

### CODE REVIEW

**Comment by Tulean** [comment]
extended check sonar

**Comment by Tulean** [comment]
extended test

**Comment by yenkins** [comment]
### Sonar scan result
 - No new bug found. Congratulation!

More detail, see in https://sonarqube-dev.intgdc.com/dashboard?id=gooddata-ui-sdk-check-extended-sonar-PR1103

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/2711/ : SUCCESS in 9s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-sonarqube-scan-zuul-el7/176/ : SUCCESS in 9m 52s (non-voting)

**Comment by yenkins** [comment]
Link to Storybook test result: https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-backstop-tests/715/artifact/libs/sdk-ui-tests/backstop/output/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/2712/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-backstop-tests/715/ : SUCCESS in 18m 07s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/2710/ : SUCCESS in 11s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/4343/ : SUCCESS in 27m 35s

**Comment by gdgate** [comment]
Build in pipeline check aborted.

File: libs/sdk-backend-tiger/src/backend/user/index.ts [hunk 1]
```diff
@@ -11,6 +12,9 @@ export class TigerUserService implements IUserService {
     }
 
     public async getUser(): Promise<IUser> {
-        throw new NotSupported("not supported");
+        return this.authCall(async (sdk) => {
+            const profile = await sdk.axios.get<IUserProfile>("/api/profile");
```
Comment (no23reason) [inline libs/sdk-backend-tiger/src/backend/user/index.ts:16]: Please add a TODO to replace this once the generated client supports this (we ideally do not want any URLs in the sdk-backend-* code).

File: libs/sdk-backend-tiger/src/utils/api.ts [hunk 2]
```diff
@@ -1,8 +1,19 @@
...
 import { TigerAuthenticatedCallGuard } from "../types";
 import { UnexpectedError } from "@gooddata/sdk-backend-spi";
 
+/**
+ * Returns organization name
+ *
+ * @public
+ */
+export const getOrganizationTitle = async (): Promise<string> => {
+    const data = (await sdk.axios.get("/api/entities/organization")).data;
```
Comment (no23reason) [inline libs/sdk-backend-tiger/src/utils/api.ts:13]: Please add a TODO to replace this once the generated client supports this (we ideally do not want any URLs in the sdk-backend-* code).

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Review by no23reason — APPROVED** [review]
LGTM

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by yenkins** [comment]
### Sonar scan result
 - No new bug found. Congratulation!

More detail, see in https://sonarqube-dev.intgdc.com/dashboard?id=gooddata-ui-sdk-gate-PR1103

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul-el7/813/ : SUCCESS in 31s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/1077/ : SUCCESS in 19s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/4355/ : SUCCESS in 19m 24s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-sonarqube-scan-zuul-el7/180/ : SUCCESS in 13m 41s (non-voting)

**Pull request merged**